### ISSUE TITLE
Indicate issue you are appealing, bullets in list not read by JAWS

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21141]
Label: 14458

    - [508-defect-4]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4

Issue Title: Indicate issue you are appealing, bullets in list not read by JAWS. (template issue)
 ## Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As with sighted users, bullets within lists help screen reader users determine where one point ends and the other begins, decreasing cognative load and increasing efficiency.
<h2>Details</h2>
On the screen where the user confirms which issue they are appealing, in the accordian which says "Don't see the issue you are appealing?" there is a list of reasons why the user's issue might not be there. Each point in this list is indicated by a black square which is read by NVDA but not read by JAWS.
Acceptance Criteria

<h2>Environment</h2>
JAWS and Edge, Chrome and Firefox.
Steps to Recreate

<h2>Proposed Solution (if known)</h2>
This looks like a template issue, as I have seen it in other areas of the site.

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs
